src/irmin-mem/irmin_mem.mli

```diff
@@ -30,7 +30,8 @@ module Atomic_write : Irmin.ATOMIC_WRITE_STORE_MAKER
 (** An in-memory store with atomic-write guarantees. *)
 
 module Make : Irmin.S_MAKER
-(** An in-memory Irmin store. *)
+(** Constructor for in-memory Irmin store. *)
 
-module KV : Irmin.KV_MAKER
-(** An in-memory KV store. *)
+(** Constructor for in-memory KV stores. Subtype of {!Irmin.KV_MAKER}. *)
```

Comment:
I'm not sure if there's a way to make this clearer in the documentation. Needing to manually expand `Irmin.KV_MAKER` is a pain.